hyperspy/io_plugins/hdf5.py

```diff
@@ -94,44 +94,47 @@ def get_hspy_format_version(f):
 
 
 def file_reader(filename, record_by, mode='r', driver='core',
```

Comment:
...maybe, but in some other PR? I personally have no clue what the `driver` does